code/datums/components/crafting/ranged_weapon.dm

```diff
@@ -296,3 +296,19 @@
 		/obj/item/storage/toolbox = 1,
 	)
 	category = CAT_WEAPON_RANGED
+
+/datum/crafting_recipe/laser_musket
+	name = "Laser Musket"
+	result = /obj/item/gun/energy/laser/musket
+	reqs = list(
+		/obj/item/weaponcrafting/stock = 1,
+		/obj/item/stack/cable_coil = 15,
+		/obj/item/stack/rods = 4,
+		/obj/item/stock_parts/micro_laser = 1,
+		/obj/item/stock_parts/capacitor = 1,
+		/obj/item/clothing/glasses/regular = 1,
+		/obj/item/reagent_containers/cup/glass/drinkingglass = 1,
+	)
+	tool_behaviors = list(TOOL_SCREWDRIVER, TOOL_WIRECUTTER)
+	time = 10 SECONDS
+	category = CAT_WEAPON_RANGED
```

Comment:
```suggestion
	category = CAT_WEAPON_RANGED
	
```

Reply:
What's changed here???

Reply:
end line. We leave an empty line at the bottom of files